core/src/main/resources/hudson/model/ComputerSet/configure.jelly

```diff
@@ -27,7 +27,15 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
-  <l:layout permission="${app.ADMINISTER}" title="${%Node Monitoring Configuration}">
+  <l:layout permission="${app.SYSTEM_READ}" title="${%Node Monitoring Configuration}">
+    <j:choose>
+      <j:when test="${app.hasPermission(app.ADMINISTER)}">
+        <j:set var="readOnlyMode" value="false" />
+      </j:when>
+      <j:otherwise>
+        <j:set var="readOnlyMode" value="true" />
```

Comment:
I wonder whether this could be simplified to 
```suggestion
        <j:set var="readOnlyMode" value="${app.hasPermission(app.ADMINISTER)}" />
```
If so, perhaps in a followup cleanup PR.